.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+> Hi there! Thanks for submitting a pull request to Concourse! To help us review your PR,
+> please fill in the following information. Feel free to delete any sections not applicable to your pull request.
+
+# Existing Issue
+> Is there an existing issue related to this PR? Fill in the issue number as follows: Fixes #1234.
+
+Fixes # .
+
+# Issue Description
```

Comment:
I find it helpful to also ask **why is this helpful** or **why do we need this PR** would allow the reviewer to reason about the use case needed from this PR.

Reply:
👍